Source/Core/Core/HW/EXI/BBA/BuiltIn.h

```diff
@@ -20,7 +20,7 @@ constexpr u16 TCP_FLAG_PSH = 0x8;
 constexpr u16 TCP_FLAG_FIN = 0x1;
 constexpr u16 TCP_FLAG_RST = 0x4;
 
-constexpr u16 IP_PROTOCOL = 0x800;
+constexpr u16 DOLPHIN_IP_PROTOCOL = 0x800;
 constexpr u16 ARP_PROTOCOL = 0x806;
```

Comment:
These values are listed at https://en.wikipedia.org/wiki/EtherType#Values - perhaps it would be better to name them `ETHERTYPE_IP` and `ETHERTYPE_ARP` (in both cases the "P" stands for protocol so `_PROTOCOL` is redundant).